sidecar/config/proxy_config.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2016 IBM Corporation
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package config
+
+import (
+	"os"
+	"strconv"
+	"text/template"
+
+	"github.com/Sirupsen/logrus"
+)
+
+type a8ServicesNginxConfData struct {
```

Comment:
I agree that in general an a8 prefix is unnecessary. In this case the a8Services prefix is specifically intended to be descriptive of the target file: amalgam8-services.conf.